packages/kbn-i18n/src/core/i18n.ts

```diff
@@ -163,6 +170,31 @@ interface TranslateArguments {
   context?: string;
 }
 
+export function isPseudoLocale() {
+  return currentLocale.toLowerCase() === PSEUDO_LOCALE;
+}
+
+export function translateUsingPseudoLocale(message: string) {
+  /**
+   * Replaces every latin char by pseudo char and repeats every third char twice.
+   */
+  function replacer() {
```

Comment:
__nit:__ once you move these functions to `./pseudo_locale.ts` please move this function out of this one - `pseudoAccentsReplacer` - we don't need to redefine it every time (even though it can be optimized by the browser).